Source/CarthageKit/FrameworkExtensions.swift

```diff
@@ -294,24 +294,24 @@ extension NSURL {
 	}
 }
 
-extension NSFileManager {
+extension FileManager {
 	/// Creates a directory enumerator at the given URL. Sends each URL
 	/// enumerated, along with the enumerator itself (so it can be introspected
 	/// and modified as enumeration progresses).
-	public func carthage_enumeratorAtURL(URL: NSURL, includingPropertiesForKeys keys: [String], options: NSDirectoryEnumerationOptions, catchErrors: Bool = false) -> SignalProducer<(NSDirectoryEnumerator, NSURL), CarthageError> {
+	public func carthage_enumerator(at url: NSURL, includingPropertiesForKeys keys: [String], options: NSDirectoryEnumerationOptions = [], catchErrors: Bool = false) -> SignalProducer<(NSDirectoryEnumerator, NSURL), CarthageError> {
```

Comment:
This could probably move onto the `Reactive` (or move into ReactiveSwift) after migration is complete.